Makefile

```diff
@@ -1,5 +1,5 @@
 DOCKER := $(shell which docker)
-HTTPS_GIT := https://github.com/cosmos/cosmos-sdk.git
+HTTPS_GIT := https://github.com/cosmos/ics23.git
```

Comment:
This change was made as the current `make proto-check-breaking` doesn't work it would say something about needing 2 contianers? I will revert this if someone can say how to make it work as intended.